config.analysis

```diff
@@ -82,6 +82,9 @@ generate = 1
 [seaice_modelvsobs]
 generate = 1
 
+[moc_postprocess]
```

Comment:
We probably do not need a specific section for mocPostprocess (it could just be 'MOC'), but I am OK with leaving this section name for now until we'll have the moc AM stuff in.